test/model/shape.spec.js

```diff
@@ -123,28 +123,6 @@
           });
           expect(shape.members.Date.timestampFormat).to.eql('iso8601');
         });
-        it('will use api metadata timestampFormat if not found elsewhere', function() {
```

Comment:
Do you know if the services that were using the timestampFormat in the metadata were using a different format than what their protocol defaulted to? If any did differ from what the protocol would require, do all of the timestamp shapes have a timestampFormat defined?

Reply:
This is a good question. Only S3 has deferent `timestampFormat` currently(`rfc822` in metadata timestamp format but `iso8601` according to this spec). But most of the timestamp members exist in headers so the timestamp format should still be `rfc822`. This only one exception is [`ResponseExpires`](https://github.com/aws/aws-sdk-js/blob/master/apis/s3-2006-03-01.min.json#L1522~L1526). Unfortunately this is a long-live bug currently, our SDK doesn't properly serialize this member so that the response can not be parsed correctly. This change fix the issue.